lms/static/scripts/frontend_apps/components/StudentSelector.js

```diff
@@ -0,0 +1,95 @@
+import { createElement } from 'preact';
+import propTypes from 'prop-types';
+
+/**
+ * A student navigation tool which shows a student selection list and a previous and next button.
+ */
+
+export default function StudentSelector({
+  onSelectStudent,
+  selectedStudentIndex,
+  students,
+}) {
+  // Disable the next button if at the end of the list. The length is equal to
+  // the student list plus the default "All Students" option.
+  const hasNextStudent = selectedStudentIndex + 1 < students.length;
+  // Disable the previous button only if the selectedStudentIndex is less than 0
+  // indicating the "All Students" choice is selected.
+  const hasPrevStudent = selectedStudentIndex >= 0;
+
+  /**
+   * Select the next student in the list.
+   */
+  const onNextStudent = () => {
+    onSelectStudent(selectedStudentIndex + 1);
+  };
+  /**
+   * Select the previous student in the list.
+   */
+  const onPrevStudent = () => {
+    onSelectStudent(selectedStudentIndex - 1);
+  };
+
+  /**
+   * Build the <select> list from the current array of students.
+   */
+  const buildStudentList = () => {
+    const options = students.map((student, i) => (
+      <option
+        key={`student-${i}`}
+        selected={selectedStudentIndex === i}
```

Comment:
`selected` is a Boolean attribute. Its presence indicates that the `option` is selected. It shouldn't take a value (its absence represents `false`). (I may have missed this on my last review—apologies!)